redbot/core/core_commands.py

```diff
@@ -2177,6 +2177,80 @@ async def command_manager(self, ctx: commands.Context):
         """Manage the bot's commands."""
```

Comment:
Considering that we can't put all these commands under `[p]cog` (😞), could you make this docstring also say "and cogs" since it's not immediately obvious?
```suggestion
        """Manage the bot's commands and cogs."""
```